perf/benchmark.jl

```diff
@@ -1,7 +1,16 @@
 import Random
 Random.seed!(1234)
 import ClimaAtmos as CA
-config = CA.AtmosCoveragePerfConfig()
+
+include("common.jl")
+
+target_job = ARGS[1]
+job_id = get(ARGS, 2, target_job)
```

Comment:
If we are using `ARGS`, I think that we should check that the correct number of arguments is passed and provide an error message explaning the various arguments. Something like:
```julia
length(ARGS) < 2 && error("Usage: benchmark.jl <target_job> <job_id>
```

Reply:
That's a good idea - I think we should just use argparse to ensure CLI input is correct since this will reduce repeated code.

Reply:
I think that this is fine. If you are using only two arguments, there's not much confusion and make things more straightforward. You also remove the dependency of this script on `ArgParse.jl` which hasn't been updated in more than 2 years.

Reply:
That sounds good to me. We could even get rid of `ArgParse` in the driver if we just take one argument.